pkg/doc/did/doc.go

```diff
@@ -762,7 +760,7 @@ func (doc *Doc) JSONBytes() ([]byte, error) {
 }
 
 // VerifyProof verifies document proofs
-func (doc *Doc) VerifyProof(suites ...verifier.SignatureSuite) error {
+func (doc *Doc) VerifyProof(suites []verifier.SignatureSuite, jsonldOpts ...jsonld.ProcessorOpts) error {
```

Comment:
Here we pass universal `jsonld.ProcessorOpts`, one of that is `WithDocumentLoader()`. The configured JSON-LD document loader with cached contexts is used in the tests, but one can pass arbitrary document loader in production code (with e.g. custom setup of HTTP client).